src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractKeyspaceIntegrationTestBase.java

```diff
@@ -22,35 +22,35 @@
 import org.junit.jupiter.api.TestInstance;
 
 /**
- * Abstract class for all int tests that needs a namespace to execute tests in. This class
- * automatically creates a namespace before all tests and drops a namespace after all test have been
+ * Abstract class for all int tests that needs a keyspace to execute tests in. This class
+ * automatically creates a keyspace before all tests and drops a keyspace after all test have been
  * run.
  *
  * <p>Note that this test uses a small workaround in {@link #getTestPort()} to avoid issue that
  * Quarkus is not setting-up the rest assured target port in the @BeforeAll and @AfterAll methods
  * (see https://github.com/quarkusio/quarkus/issues/7690).
  */
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
-public abstract class AbstractNamespaceIntegrationTestBase {
+public abstract class AbstractKeyspaceIntegrationTestBase {
```

Comment:
idea: to reduce size of this PR, could create separate PR that only renames this test base class and references to `namespaceName` (to `keyspace`) -- this is internal, non-user-visible change, and could be merged before any other changes. But would remove dozens of files where the only change is for naming.

Reply:
Thanks, let me separate from this PR.